app/locales/en/translations.js

```diff
@@ -248,6 +248,11 @@ export default {
             support: `Support`,
             contact: `Contact`
         },
+        'permission-language':{
```

Comment:
This needs to be collapsed. We don't want the translations to be duplicated. We can add a configuration flag in config/environment.js to conditionally include the second sentence.